Python/ceval.c

```diff
@@ -1064,9 +1064,11 @@ _PyEval_EvalFrameDefault(PyFrameObject *f, int throwflag)
            Py_MakePendingCalls() above. */
 
         if (_Py_atomic_load_relaxed(&eval_breaker)) {
-            if (_Py_OPCODE(*next_instr) == SETUP_FINALLY) {
+            if (_Py_OPCODE(*next_instr) == SETUP_FINALLY
```

Comment:
Yeah, any compiler will trivially handle this, so I'd rather keep the more straightforward code.

gcc and clang both produce identical code for the local variable and no-local-variable code on `-O1` and higher: https://godbolt.org/g/AJY9Zi